docs-src/cloud/available-regions.md

```diff
@@ -9,7 +9,17 @@ tags:
   - explanation
 ---
 
-Temporal Cloud currently runs in the following regions in Amazon Web Services (AWS):
+**Where is Temporal Cloud available?**
+
+Developers and applications can access Temporal Cloud from any location with internet connectivity, irrespective of where the Temporal Cloud resources (Namespaces) are located.
+
+Temporal Cloud is compatible with applications deployed in various cloud environments or data centers.
+
+_**Your Workers and Client code aren't required to be hosted on AWS.**_
```

Comment:
This should probably get reordered to below the mention of AWS regions, right now it feels a bit out of pocket